ksuid/ksuid.py

```diff
@@ -116,7 +116,7 @@ class KsuidMs(Ksuid):
     # Timestamp is a uint32
     TIMESTAMP_LENGTH_IN_BYTES = 5
 
-    # Payload is 16-bytes
+    # Payload is 15-bytes
```

Comment:
This and other comments and arguably redundant. Another option is removing them.

Reply:
I'm happy with removing TBH, but I think it's fine to keep.